_includes/header.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-expand-lg navbar-light">
   <!-- Group Logo can go here -->
   <a href="/">
-    <img src="../img/PyHC logo 250x250.png" alt="PyHC Logo" width="52" height="52">
+    <img src="{{ site.baseurl }}/img/PyHC logo 250x250.png" alt="PyHC Logo" width="52" height="52">
```

Comment:
Apparently the file structure changes for the secondary blog pages such that `../img/` wasn't a valid directory for them; the logo at the top was appearing as a square with a question mark. Referencing `site.baseurl` instead fixed it.

Didn't notice this until a few minutes ago.